website/react-magma-docs/src/pages/design/indeterminate.mdx

```diff
@@ -9,69 +9,13 @@ import { LeadParagraph } from '../../components/LeadParagraph';
 
 <PageContent componentName="indeterminate_checkbox" type="design">
 
-<LeadParagraph>
-  Selection controls allow the user to select options.
-</LeadParagraph>
-
```

Comment:
The spacing on the page gets a little odd when it doesn't start with a LeadParagraph. Fine to leave it out if it doesn't bother you, but would recommend keeping it for consistency.

Reply:
Fair point. I found some good text to put in place there.